### ISSUE TITLE
alba-core basecalling  causes assertion to faill

### ISSUE THREAD

**Issue by bernhard314** [jts/nanopolish#93]
Hi Jared (and team),

using the local alba-core basecalling results caused an assertion to fail in eventalign. 

```
nanopolish eventalign -t 8 --sam -r reads.fa -b reads.sorted.bam -g Ecoli.fasta --models nanopolish_models.fofn > evtalgn.tsv
```

```
nanopolish: src/nanopolish_squiggle_read.cpp:304: std::vector<EventRangeForBase> SquiggleRead::build_event_map_1d(const string&, uint32_t, std::vector<fast5::Event_Entry>&): Assertion `f5_events[0].move == 0' failed.
Aborted (core dumped)

```
Read-quality was good and nanopolish eventalign worked with the online basecalling results.
There are quite a few differences between the rnn and the online basecalling output. 
We looked into the code and tried trimming the alba-core results in the fast5 files, but did not find a solution.
Is there a workaround?

Bernhard

**Comment by jts** [comment]
Hi,

Could you send me (by email) a fast5 file that demonstrates the problem?

Thanks,
Jared

**Comment by damientully** [comment]
Hi Bernhard & Jared

I'm experiencing the same problem with local albacore data. Any solution to this problem?

Thanks,
Damien

**Comment by jts** [comment]
Fixed in 0.6
